snuba/consumers/consumer_builder.py

```diff
@@ -237,11 +283,52 @@ def log_general_error(e: KafkaError) -> None:
 
         return StreamProcessor(consumer, self.raw_topic, strategy_factory, IMMEDIATE)
 
+    # We currently don't having slicing implemented for multistorage case
```

Comment:
I suggest you immediately throw an error if the user tries to pass a slice with multiple storages, so it is very clear that combination is not support, rather than just relying on someone reading comment to figure out it doesn't work

Reply:
In the new refactor, `build_multistorage_streaming_strategy_factory` is in multistorage_consumer.py, with defined args, so someone can't pass in slice_id.